crml/nft/src/types.rs

```diff
@@ -341,6 +348,17 @@ pub type TokenCount = SerialNumber;
 /// Global unique token identifier
 pub type TokenId = (CollectionId, SeriesId, SerialNumber);
 
+// A value placed in storage that represents the current version of the NFT storage. This value
```

Comment:
This is clever, I like the intent

Reply:
this reminds me I needed to update the storage version after updating